widget/embedded/src/components/Slippage/Slippage.styles.ts

```diff
@@ -10,6 +10,7 @@ export const SlippageChipsContainer = styled('div', {
   flexWrap: 'wrap',
   gap: '$10',
   [`& ${TextField}`]: {
+    height: '$32',
```

Comment:
```suggestion
    boxSizing: 'content-box',
```
I think it's better to handle element taking extra space using `boxSizing` instead of adding a magical number for height.

Reply:
I was setting a fixed height on the input to prevent layout shifts when displaying success or error states. This was because the border was conditionally applied with different thicknesses (e.g., 1px), which caused the input’s height to increase.
Now, I’ve replaced that workaround with a cleaner solution: using a consistent border of `1px solid transparent` and only changing the `border-color` based on the input state. This way, the input always keeps the same dimensions, and we avoid unnecessary height changes without hardcoding the height.